docs/import/center.js

```diff
@@ -1,55 +1,47 @@
 import {disasterDocumentReference} from '../firestore_document.js';
 import {convertEeObjectToPromise} from '../map_util.js';
 
-export {storeCenter as default};
-
-/**
- * Adds the lat and lng of a feature's centroid as properties.
- * @param {ee.Feature} feature
- * @return {ee.Feature}
- */
-function withGeo(feature) {
-  const centroid = feature.centroid().geometry().coordinates();
-  return feature.set('lng', centroid.get(0), 'lat', centroid.get(1));
-}
+export {computeAndSaveBounds, saveBounds};
 
 /**
  * Stores an approximate bounds around a given feature collection.
  *
  * @param {ee.FeatureCollection} features the featureCollection around which you
  * want to orient the map
- * @param {Promise} firebaseAuthPromise
  * @return {Promise<Array<number>>} Promise that completes with array of bounds
  * after Firestore write is complete.
  */
-function storeCenter(features, firebaseAuthPromise) {
-  const damageWithCoords = ee.FeatureCollection(features.map(withGeo));
-  // This is assuming we're not crossing the international date line...
-  const outerBounds = ee.List([
-    damageWithCoords.aggregate_min('lat'),
-    damageWithCoords.aggregate_min('lng'),
-    damageWithCoords.aggregate_max('lat'),
-    damageWithCoords.aggregate_max('lng'),
-  ]);
-  return Promise
-      .all([firebaseAuthPromise, convertEeObjectToPromise(outerBounds)])
-      .then((results) => saveBounds(results[1]));
+function computeAndSaveBounds(features) {
+  return convertEeObjectToPromise(features.geometry().bounds())
```

Comment:
is features.geometry a no no?

Reply:
This is way faster than the previous method. I agree we were told not to use FeatureCollection.geometry(), but the reason was that it's a huge bag of geometries. However, once you do .bounds(), it's just a rectangle. So not a lot of data.